todoman/ui.py

```diff
@@ -100,6 +99,15 @@ def __init__(self, todo, lists, formatter):
                         in widgets.ExtendedEdit.HELP)
         )
 
+        def change_current_list(radio_button, new_state, new_list):
+            self.current_list = new_list
+
+        for _list in self.lists:
+            urwid.RadioButton(items, _list.name,
+                              state=_list is self.current_list,
```

Comment:
Sorry, this actually needs to be `_list.name == self.current_list.name`. Also you will hit an error because not everything in `items` is a RadioButton. For that reason, please create a new list `list_selector = []`, pass that instead of `items` to RadioButton, then after the for-loop:

    items.append(urwid.Pile(list_selector))